### PR TITLE
Implemented cli command structure

### PR DESCRIPTION
**Pull request by KairaMiu**
Commands only execute dummy actions (KYT-156).

### CODE REVIEW

File: kyt/cmd/root.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
```
Comment (KlausPopp) [inline kyt/cmd/root.go:1]: Remove the toggle flag

File: kyt/dobi.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 mount=mount-kyt-src:
   bind: "kyt/"
   path: "/home/user/src"
-  read-only: true
+  read-only: false
```
Comment (KlausPopp) [inline kyt/dobi.yaml:7]: Why has this been change to false? It should be true

**Review by KlausPopp — CHANGES_REQUESTED** [review]

File: kyt/cmd/root.go (hunk 1 above)
Comment (KairaMiu) [reply kyt/cmd/root.go:1]: Done.

File: kyt/dobi.yaml (hunk 2 above)
Comment (KairaMiu) [reply kyt/dobi.yaml:7]: Go modules can be added during development process. As dobi is a development tool, it is required that dobi can update go.mod file.

Comment (KlausPopp) [reply kyt/dobi.yaml:7]: Ok

**Review by siredmar — APPROVED** [review]

**Pull request merged**